app/controllers/concerns/attendance_concerns.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module AttendanceConcerns
+  extend ActiveSupport::Concern
+
+  def attending_workshops
+    current_user.nil? ? Set.new : current_user.workshop_invitations.accepted.pluck(:id).to_set
```

Comment:
Micro detail: the `ids` method - https://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-ids